### PR TITLE
DAH-1142 Uptake new listing card redesign

### PR DESCRIPTION
**Pull request by emilyjablonski**
[DAH-1142](https://sfgovdt.jira.com/browse/DAH-1142?atlOrigin=eyJpIjoiMTQ4MGI2MzdiODAxNDQxNWI4MjE4ZDQxZjI1ZTcwYzEiLCJwIjoiaiJ9)

Uptakes the latest UI components including the breaking changes around the new ListingCard redesign.

Yindi has done a separate design review, but an overview of the visual changes are: 
The listing name and address is moved off the image so that we can remove the image overlay, there are new flexible tag and table header props, and the details link aligns right.

I also had to manually add the header to the Habitat section since that content is custom.

Screen Shot 2022-04-05 at 3 10 21 PM

### CODE REVIEW

File: app/javascript/modules/listings/DirectoryHelpers.tsx [hunk 1]
```diff
@@ -185,14 +185,16 @@ export const getListingCards = (listings, directoryType, stackedDataFxn, hasFilt
     return (
       <ListingCard
         key={index}
+        stackedTable={true}
```
Comment (jacksongonzales) [inline app/javascript/modules/listings/DirectoryHelpers.tsx:188]: this is probably just a convention nitpick, but I'm used to passing fixed boolean props with presence/absence of the attribute. does TypeScript change this though? 🤔

File: app/javascript/modules/listings/HabitatForHumanity.tsx [hunk 2]
```diff
@@ -25,21 +26,32 @@ export const getHabitatContent = (listing, stackedDataFxn) => {
...
+          )
+        )}
 
-      {getHeader(t("listings.habitat.payments"))}
-      <div>{t("listings.habitat.payments.desc")}</div>
-      <div className={"mt-4  border-0 border-b pb-2 mb-2"}>{t("listings.habitat.infoSession")}</div>
-    </div>
+        {getHeader(t("listings.habitat.payments"))}
+        <div>{t("listings.habitat.payments.desc")}</div>
+        <div className={"mt-4  border-0 border-b pb-2 mb-2"}>
```
Comment (jacksongonzales) [inline app/javascript/modules/listings/HabitatForHumanity.tsx:51]: another nitpick 🤪 
```suggestion
        <div className={"mt-4 border-0 border-b pb-2 mb-2"}>
```

**Review by jacksongonzales — APPROVED** [review]
left a few style comments, looks so much better without the text over images!!

File: app/javascript/modules/listings/DirectoryHelpers.tsx (hunk 1 above)
Comment (emilyjablonski) [reply app/javascript/modules/listings/DirectoryHelpers.tsx:188]: Just a personal preference, I don't mind either way, but this seems to be how we're doing it elsewhere - super down to update that style

**Review by christine-sfg — APPROVED** [review]

**Pull request merged**